tests/admin-mode_test.go

```diff
@@ -0,0 +1,270 @@
+			     s/applabel: 'app=nginx'/applabel: 'run=adminapp'/g`,
+				experimentName+"-ce.yaml").Run()
+			//Modify FORCE
+			err = exec.Command("sed", "-i", `/name: FORCE/{n;s/.*/              value: ""/}`, experimentName+"-ce.yaml").Run()
+			Expect(err).To(BeNil(), "Failed to Modify FORCE field of chaos engine")
+
+			//Creating ChaosEngine
+			By("Creating ChaosEngine")
+			err = exec.Command("kubectl", "apply", "-f", experimentName+"-ce.yaml", "-n", chaosTypes.ChaosNamespace).Run()
+			Expect(err).To(BeNil(), "Failed to create chaos engine")
+			fmt.Println("ChaosEngine for pod delete created successfully")
+			time.Sleep(2 * time.Second)
+
+			//Fetching the runner pod and Checking if it gets in Running state or not
+			By("Wait for runner pod to come in running sate")
+			runnerNamespace := chaosTypes.ChaosNamespace
+			runnerPodStatus, err := utils.RunnerPodStatus(runnerNamespace, engineName, client)
+			Expect(runnerPodStatus).NotTo(Equal("1"), "Faileded to get in running state")
+			Expect(err).To(BeNil(), "Failed to get the runner pod")
+			fmt.Println("Runner pod for pod delete is in Running state")
+
+			//Waiting for experiment job to get completed
+			//Also Printing the logs of the experiment
+			By("Waiting for job completion")
+			jobNamespace := chaosTypes.ChaosNamespace
+			jobPodLogs, err := utils.JobLogs(experimentName, jobNamespace, engineName, client)
+			Expect(jobPodLogs).To(Equal(0), "Failed to print the logs of the experiment")
+			Expect(err).To(BeNil(), "Failed to get the experiment job pod")
+
+			//Checking the chaosresult
+			By("Checking the chaosresult")
+			chaosResult, err := clientSet.ChaosResults(chaosTypes.ChaosNamespace).Get(engineName+"-"+experimentName, metav1.GetOptions{})
+			Expect(string(chaosResult.Status.ExperimentStatus.Verdict)).To(Equal("Pass"), "Verdict is not pass chaosresult")
+			Expect(err).To(BeNil(), "Failed to get chaosresult")
+		})
+	})
+	// BDD TEST CASE 2
+	// Operator in litmus ns, engine and experiment is in test ns
+	// Application is in deafult ns
+	Context("Check for litmus components", func() {
+
+		It("Should check for creation of runner pod", func() {
+
+			//Create Namespace for the test
+			By("Creating namespace for the test")
+			cmd := exec.Command("kubectl", "create", "ns", "test")
+			cmd.Stderr = os.Stderr
+			err := cmd.Run()
+			if err != nil {
+				log.Fatalf("cmd.Run() failed with %s\n", err)
+			}
+
+			//Installing RBAC for experiment that is pod-delete
+			rbacPath := "https://raw.githubusercontent.com/litmuschaos/pages/master/docs/litmus-admin-rbac.yaml"
+			err = exec.Command("wget", "-O", experimentName+"-sa.yaml", rbacPath).Run()
+			Expect(err).To(BeNil(), "Fail to fetch rbac")
+			err = exec.Command("sed", "-i", `s/namespace: litmus/namespace: test/g`, experimentName+"-sa.yaml").Run()
+			Expect(err).To(BeNil(), "Fail to modify rbac")
+			err = exec.Command("kubectl", "apply", "-f", experimentName+"-sa.yaml").Run()
```

Comment:
this should be in test ns?

Reply:
in yaml it has been changed to test namespace so it will create there only?